docs/dev/spatial-data.md

```diff
@@ -0,0 +1,41 @@
+# Spatial data
+
+We store certain geospatial data in our own database and use it for some basic
+geospatial queries. In order for the site to work correctly locally, the data
+needs to be loaded first. At the moment, you will need to fetch two shapefiles
+from NWS and the Cities500 dataset from geonames.
+
+1. Download the U.S. Counties and U.S. States and Territories shapeiles from the
```

Comment:
I assumed .gitignore already ignores these?

Reply:
It does now! 😛 Thank you for catching that.